### DISCUSSION TITLE
Is web already supported?

### DISCUSSION THREAD

**Discussion by busslina** [GregoryConrad/mimir#302] [General]
![imagen](https://github.com/GregoryConrad/mimir/assets/73592852/52f4d5e6-ba21-4711-9343-5991a9cf2ca4)

![imagen](https://github.com/GregoryConrad/mimir/assets/73592852/ab98ef89-5939-418c-aba4-1bdd03d6f372)

**Comment by GregoryConrad** [comment]
No, unfortunately not and I don’t have the time to properly do it anytime soon

**Comment by busslina** [comment]
Thanks!
